src/main/java/uk/gov/gchq/palisade/resource/AbstractLeafResource.java

```diff
@@ -111,16 +133,8 @@ public AbstractLeafResource parent(final ParentResource parent) {
         return this;
     }
 
-    @Override
-    public ParentResource getParent() {
-        requireNonNull(parent, "The parent has not been set for this resource.");
-        return parent;
-    }
 
-    @Override
-    public void setParent(final ParentResource parent) {
-        parent(parent);
-    }
+
 
     @Override
     public boolean equals(final Object o) {
```

Comment:
Update to the new equals, hashCode and toString methods

Reply:
Ok, changed all of them.